src/main/webapp/templates/profileBuilder.html

```diff
@@ -9,7 +9,67 @@
   <link href="css/styles.css" rel="stylesheet">
 </head>
 <body>
+    <h1>Profile Builder</h1>
+        <form action="/profile" id="profile_form" method="POST" id="makeProfile">
+            <div>
+                <label style= "color: black;"> User Type:
+                <select class="" placeholder={{category}} name="user-Type" onchange="setForm(this.value)">
```

Comment:
Is the empty class attribute needed (L16)?

Reply:
No we don't. K done. Thanks for adding lines!